specification/resources/resource-manager/Microsoft.Resources/stable/2022-12-01/subscriptions.json

```diff
@@ -0,0 +1,874 @@
+            "Recommended",
+            "Extended",
+            "Other"
+          ],
+          "x-ms-enum": {
+            "name": "RegionCategory",
+            "modelAsString": true
+          }
+        },
+        "geography": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The geography of the location."
+        },
+        "geographyGroup": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The geography group of the location."
+        },
+        "longitude": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The longitude of the location."
+        },
+        "latitude": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The latitude of the location."
+        },
+        "physicalLocation": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The physical location of the Azure location."
+        },
+        "pairedRegion": {
+          "type": "array",
+          "description": "The regions paired to this region.",
+          "items": {
+            "$ref": "#/definitions/PairedRegion"
+          }
+        },
+        "homeLocation": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The home location of an edge zone."
+        }
+      }
+    },
+    "Location": {
+      "properties": {
+        "id": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The fully qualified ID of the location. For example, /subscriptions/00000000-0000-0000-0000-000000000000/locations/westus."
+        },
+        "subscriptionId": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The subscription ID."
+        },
+        "name": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The location name."
+        },
+        "type": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The location type.",
+          "enum": [
+            "Region",
+            "EdgeZone"
+          ],
+          "x-ms-enum": {
+            "name": "LocationType",
+            "modelAsString": false
+          }
+        },
+        "displayName": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The display name of the location."
+        },
+        "regionalDisplayName": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The display name of the location and its region."
+        },
+        "metadata": {
+          "description": "Metadata of the location, such as lat/long, paired region, and others.",
+          "$ref": "#/definitions/LocationMetadata"
+        },
+        "availabilityZoneMappings": {
+          "type": "array",
+          "description": "The availability zone mappings for this region.",
+          "items": {
```

Comment:
It is recommended to define "x-ms-identifiers" property to indicate identifier of objects in the array. See [https://github.com/Azure/azure-openapi-validator/blob/main/docs/xms-identifier-validation.md](https://github.com/Azure/azure-openapi-validator/blob/main/docs/xms-identifier-validation.md). For example, here, if logicalZone or physicalZone will be unique for each object in array, that can be set as "x-ms-identifiers".